examples/operator.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: etcd-cluster-operator
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  namespace: default
+  name: configmap-reader
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get", "create", "watch", "update"]
+  - apiGroups: [""]
+    resources: ["services"]
+    verbs: ["create", "get", "list", "watch"]
+  - apiGroups: ["apps"]
+    resources: ["replicasets"]
+    verbs: ["list", "get", "create", "watch"]
+  - apiGroups: ["etcd.improbable.io"]
+    resources: ["etcdpeers"]
+    verbs: ["list", "get", "watch"]
+  - apiGroups: ["etcd.improbable.io"]
+    resources: ["etcdclusters"]
+    verbs: ["list", "get", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: etcd-cluster-operator-configmap-reader
+  namespace: default
+subjects:
+  - kind: ServiceAccount
+    name: etcd-cluster-operator
+    namespace: default
+roleRef:
+  kind: ClusterRole
+  name: configmap-reader
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: etcd-cluster-operator
+  name: etcd-cluster-operator
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: etcd-cluster-operator
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: etcd-cluster-operator
+    spec:
+      containers:
+      - args:
+        - --enable-leader-election
+        command:
+        - /manager
+        image: etcd-cluster-operator:test
+        imagePullPolicy: IfNotPresent
```

Comment:
Should be `Never` I'd guess, given the `image`?

Reply:
```suggestion
        imagePullPolicy: Never
```